frontend/src/pages/Overview/__tests__/OverviewPage.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('Overview page', () => {
 
   it('renders initial layout', () => {
     const wrapper = shallow(
-      <OverviewPage meshStatus={MTLSStatuses.NOT_ENABLED} navCollapse={false} duration={600} refreshInterval={10000} kiosk={''} minTLS={''} istioAPIEnabled={false}/>
+      <OverviewPage meshStatus={MTLSStatuses.NOT_ENABLED} navCollapse={false} duration={600} refreshInterval={10000} kiosk={''} minTLS={''} istioAPIEnabled={false} isMaistra={false}/>
```

Comment:
You lost the separator above `Remove Traffic Policies` in the Maistra use case.

Reply:
I don't understand what this is commenting on. This is a Cypress test and it passed when GitHub Actions ran the tests. Can you be more specific what this code here on line 104 needs to do?

Reply:
I submitted a new commit that retains the separator line. See the latest screen captures in my latest comment.

Reply:
I just added the comment in the wrong place.  As long as the separator is back that's good.